repos/actors/checkuptime/libraries/checkuptime.py

```diff
@@ -0,0 +1,27 @@
+import os
+import time
+
+from leapp import reporting
+from leapp.libraries.stdlib import api
+
+
+def check_excessive_uptime():
+    # TODO: can this be parametrized?
+    excessive_uptime_limit = 30 * 24 * 60 * 60  # 30 days in seconds
+
+    with open("/proc/uptime", "r") as uptime_file:
+        uptime_seconds = float(uptime_file.readline().split()[0])
+
+    if uptime_seconds > excessive_uptime_limit:
+        summary = (
+            "Excessive uptime detected: {:.0f} days. "
+            "Consider rebooting the system before proceeding with the upgrade."
+        ).format(uptime_seconds / (24 * 60 * 60))
+
+        report = [
+            reporting.Title("Excessive uptime detected"),
+            reporting.Summary(summary),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Groups([reporting.Groups.SANITY, reporting.Groups.INHIBITOR]),
+        ]
```

Comment:
The report seems to me weird.
* I am missing explanation why this should be problem. The summary is actualy leaving me in blind (I consider this pretty common to see a system that is running for longer than 1month). 
* Also reading the summary, I would understand the msg as purely informative, however then I will realize i has high severity and it's inhibitor so it blocks the upgrade
* From the msg, I do not have idea what is actually the excessive uptime. Where is the limit? How this limit has been set?
* `Consider rebooting the system before proceeding with the upgrade.` is not summary, it's remediation hint. In this case it's written in the way that people could do it if they want. But the inhibitor says they must do it.

Reply:
Thanks for the feedback! The previous commit just outlined the report. I incorporated [Bob's comment](https://github.com/oamg/leapp-supplements/pull/1#issuecomment-1506887106) into the report messages now.

I listed the mismatched kernel versions and modified /boot files as related sources instead of putting them into the entry summary. Is that expected?